eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -239,9 +239,9 @@ static function (Type $type) use ($status, $getTypeKeysFn) {
     public function loadByIdentifier($identifier)
     {
         return $this->getCacheValue(
-            $identifier,
+            $this->escapeForCacheKey($identifier),
             'ez-content-type-',
-            function ($identifier) {
+            function () use ($identifier) {
```

Comment:
Is this change necessary?

Reply:
without this change, we will use the variable after using the escapeForCacheKey function, and we need the original value to search in the database